cmd/cli/uninstall_mesh.go

```diff
@@ -171,6 +171,17 @@ func (d *uninstallMeshCmd) run() error {
 			}
 
 			if err == nil {
+				namespaces, _ := selectNamespacesMonitoredByMesh(m.name, d.clientSet)
+				for _, ns := range namespaces.Items {
+					removeCmd := &namespaceRemoveCmd{
+						out:       d.out,
+						namespace: ns.Name,
+						meshName:  m.name,
+						clientSet: d.clientSet,
+					}
+
+					_ = removeCmd.run()
```

Comment:
@mudit-01 I see in a previous commit you did an error check on this command. Is there a reason this was removed? I think you were on the right track adding the error! I think it would be good to add a print statement to indicate that the namespace cleanup failed. Thank you!
```suggestion
					if err := removeCmd.run(); err != nil {
					    fmt.Fprintf("Could not cleanup OSM mesh [%s] related labels and annotations for namespace  [%s]....", ...)
					}
```